1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,7 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+let groupIsOnlyStudents = group.every((i) => students.includes(i) ); // complete this statement
```

Comment:
Try to use meaning full parameters! instead of i you can use group.every((student) => students.includes(student))